content/applications/sales/sales/products_prices/prices/pricing.rst

```diff
@@ -1,146 +1,439 @@
+:guilabel:`Pricelists` setting chosen: either :guilabel:`Multiple prices per product` or
+:guilabel:`Advanced price rules (discounts, formulas)`.
+
+However, the :guilabel:`Time-based rules` tab and :guilabel:`Configuration` tab are always the same,
+regardless of the chosen :guilabel:`Pricelists` setting.
+
+Price rules tab (multiple prices per product)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+With the :guilabel:`Multiple prices per product` setting activated, the :guilabel:`Price Rules` tab
+on pricelist forms provides the option to add specific products, with a specific price, to a
+pricelist.
+
+To add a specific product and price to a pricelist form, click the :guilabel:`Price Rules` tab, then
+click :guilabel:`Add a line` in the :guilabel:`Products` column. Then, select the desired product
+for which a specific price should be applied.
+
+Next, if necessary, select a product variant under the :guilabel:`Variants` column (e.g. a specific
+product size, color, etc.). If no variants are selected, then this price will apply to all variants
+of the product.
+
+Under the :guilabel:`Min. Quantity` column, designate if there should be a minimum amount of the
+product purchased, in order to apply the specific price.
 
-Several prices per product
-==========================
+To configure the price of the product for this specific pricelist, enter the desired amount under
+the :guilabel:`Price` column. Then, there is the option to add a :guilabel:`Start Date` and
+:guilabel:`End Date` to the configured product price, if desired.
 
-To apply several prices per product, select *Different prices per customer
-segment* in :menuselection:`Sales --> Settings`. Then open the *Sales* tab
-in the product detail form. You can settle following strategies.
+To add another product line, click :guilabel:`Add a line` again, and repeat the process. There is no
+limit to how many products can be added in the :guilabel:`Price Rules` tab of a pricelist form.
 
-Prices per customer segment
----------------------------
+For more information, check out the following section:
+:ref:`Multiple prices per product <sales/multiple-prices-per-product>`
 
-Create pricelists for your customer segments: e.g. registered, premium, etc.
+Price rules tab (advanced price rules)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. image:: pricing/pricing_customer.png
+With the :guilabel:`Advanced price rules (discounts, formulas)` setting activated, the
+:guilabel:`Price Rules` tab on pricelist forms provides the option to configure detailed price rules
+based on formulas.
+
+Check out the :ref:`Advanced price rules (discounts, formulas) <sales/advanced-price-rules>` section
+for detailed steps on how to add advanced price rules to a pricelist.
+
+Time-based rules tab
+--------------------
+
+Time-based rules are used specifically with :doc:`subscription products
+</applications/sales/subscriptions/products>`. Be sure to check out the Odoo *Subscriptions*
+documentation.
```

Comment:
optional– I think it would be good to link to the top-level Subscriptions documentation if referencing it :)
```suggestion
:doc:`documentation </applications/sales/subscriptions>`.
```